selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -378,6 +378,51 @@ public String getVisibleTextFromEditor() {
     return getTextFromOrionLines(lines);
   }
 
+  /**
+   * Gets visible text from specified {@code line}.
+   *
+   * @param editorLine number of line for getting text
+   * @return visible text from specified line
+   */
+  public String getVisibleText(int editorLine) {
+    if (0 == editorLine) {
+      String errorMessage =
+          format(
+              "Specified line: \"%s\" does not exist. The editor numeration starts from \"1\". "
+                  + "Please specify correct line.",
+              editorLine);
+      throw new ArrayIndexOutOfBoundsException(errorMessage);
+    }
+
+    final int lineIndex = editorLine - 1;
```

Comment:
Could be useful as a separate method of CodenvyEditor: `getVisibleLineFromEditor(lineNumber)`.